.github/workflows/tests.yaml

```diff
@@ -77,6 +77,7 @@ jobs:
           pytest \
             --timeout=9 \
             --durations=10 \
+            --cov-fail-under=35 \
```

Comment:
I will increase this as I go, but I'd like to set up required checks going forward.